src/main/java/com/google/firebase/database/utilities/encoding/CustomClassMapper.java

```diff
@@ -498,7 +493,7 @@ public BeanMapper(Class<T> clazz) {
             if (existingPropertyName != null) {
               if (!existingPropertyName.equals(propertyName)) {
                 throw new DatabaseException(
-                    "Found setter with invalid case-sensitive name: " + method.getName());
+                    "Found setter with invalid " + "case-sensitive name: " + method.getName());
```

Comment:
Seems unnecessary.

Reply:
I removed this instance and the remaining string concatenations in this file that follow a similar pattern.

Reply:
Thanks 👍